chainntnfs/interface_test.go

```diff
@@ -412,13 +412,172 @@ func testMultiClientConfirmationNotification(miner *rpctest.Harness,
 	}
 }
 
+// Tests the case in which a confirmation notification is requested for a
+// transaction that has already been included in a block. In this case,
+// the confirmation notification should be dispatched immediately.
+func testTxConfirmedBeforeNtfnRegistration(miner *rpctest.Harness,
+	notifier chainntnfs.ChainNotifier, t *testing.T) {
+
+	t.Logf("testing transaction confirmed before notification registration")
+
+	// First, let's send some coins to "ourself", obtainig a txid.
+	// We're spending from a coinbase output here, so we use the dedicated
+	// function.
+
+	txid, err := getTestTxId(miner)
+	if err != nil {
+		t.Fatalf("unable to create test tx: %v", err)
+	}
+
+	// Now generate two blocks. The transaction will be included in the
```

Comment:
The comment and the code are out of sync here, it says two blocks, but only on block is actually generated. It doesn't seem to affect the correctness of the test, so this is a minor nit.